backend/src/test/java/com/allog/dallog/domain/member/application/MemberServiceTest.java

```diff
@@ -138,51 +138,159 @@ class MemberServiceTest extends ServiceTest {
         assertThat(actual.getDisplayName()).isEqualTo(패트_이름);
     }
 
-    @DisplayName("회원을 삭제한다.")
+    @DisplayName("회원 탈퇴를 한다.")
     @Test
-    void 회원을_삭제한다() {
+    void 회원_탈퇴를_한다() {
         // given
-        Long 후디_id = parseMemberId(AuthFixtures.후디_인증_코드_토큰_요청());
+        Member 관리자 = memberRepository.save(관리자());
+        Category 내_일정 = categoryRepository.save(내_일정(관리자));
+        CategoryRole 역할 = categoryRoleRepository.save(new CategoryRole(내_일정, 관리자, CategoryRoleType.ADMIN));
+        Subscription 내_일정_구독 = subscriptionRepository.save(new Subscription(관리자, 내_일정, Color.COLOR_1));
 
         // when
-        memberService.deleteById(후디_id);
+        memberService.deleteById(관리자.getId());
 
         // then
-        assertThatThrownBy(() -> memberService.findById(후디_id))
+        assertThatThrownBy(() -> memberService.findById(관리자.getId()))
                 .isInstanceOf(NoSuchMemberException.class);
     }
 
-    @DisplayName("회원을 삭제할때, 관련된 데이터도 모두 삭제한다.")
+    @DisplayName("회원 탈퇴를 할 때, 다른 회원이 만든 카테고리를 구독만 하면 정상적으로 탈퇴한다.")
```

Comment:
r: MemberServiceTest의 전체적인 테스트 메서드 이름을 이런 형식으로 바꿔보면 어떨까요?  
```suggestion
    @DisplayName("다른 회원이 만든 카테고리를 구독만 한 경우 탈퇴할 수 있다.")
```
```
    @DisplayName("구독한 카테고리의 유일한 관리자가 아닌 경우 탈퇴할 수 있다.")
    @DisplayName("구독한 카테고리의 관리자가 더 있고 권한이 없는 구독 카테고리가 있는 경우 탈퇴할 수 있다.")
```
이런 식으로요! `~할 때`를 빼도 좋을 것 같아요. 그리고 enum 이름을 그대로 쓰는 경우엔 enum 이름이 바뀌면 테스트 네임까지 다 바뀌어야 할 것 같아서 한글로 풀어 써주는 게 좋을 것 같아요 ;_;

`@백엔드 전부` 추가적으로 member를 '유저', '회원', '멤버' 등으로 부르고 있네요 ㅠ_ㅠ 용어 통일이 필요해보입니다!!!